README.adoc

```diff
@@ -0,0 +1,63 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
+
+This repository contains add-ons contributed by the Minishift community.
+
+We hope that by sharing these add-ons, folks in the community will be able to make the most out of their Minishift environment.
+Community add-ons are tested and supported by the community.
+
+[[introduction-addons]]
+== Introduction to add-ons
+
+link:https://docs.openshift.org/latest/minishift/using/addons.html[Minishift add-ons] are extensions, or plug-ins, that customize the default OpenShift cluster configuration and behavior.
+For example, you can use an add-on to allow authenticated users to run images without a pre-allocated UID, or to run commands on the Docker daemon inside the Minishift VM.
+
+Minishift includes several link:https://docs.openshift.org/latest/minishift/using/addons.html#default-addons[default add-ons], which are stored in *_$MINISHIFT_HOME/addons_*.
+These add-ons are tested and supported by the Minishift core team, although due to the experimental nature of the add-ons mechanism, backwards-compatibility is not guaranteed.
```

Comment:
Can we please remove ", although due to the experimental nature of the add-ons mechanism, backwards-compatibility is not guaranteed."